network/benches/network_bench.rs

```diff
@@ -7,6 +7,8 @@
 #![allow(clippy::trivially_copy_pass_by_ref)]
 // Allow writing 1 * KiB or 1 * MiB
 #![allow(clippy::identity_op)]
+// Criterion API has changed, TODO: Remove parameterized groups, and bench()
+#![allow(deprecated)]
 
 use bytes::Bytes;
 use criterion::{
```

Comment:
Okay cool, I was debating removing the benches altogether given I don't know if they're even up to date anymore.